extensions/src/main/java/org/pentaho/platform/plugin/action/ActionInvokerSystemListener.java

```diff
@@ -202,7 +220,12 @@ private void buildPayload( String urlEncodedJson, String enc ) {
     }
 
     IActionInvokeStatus issueRequest( ) throws Exception {
+      publishWorkItemStatus( WorkItemLifecyclePhase.RECEIVED, null );
```

Comment:
We can't log the RECEIVED log until we have all the necessary pieces, and we don't have those pieces until the payload is built - I supposed we could move this call to the Payload constructor, it just seemed like this is a more appropriate place for this.